components/Days.vue

```diff
@@ -67,11 +67,13 @@ export default {
   display: none;
 }
 .days {
-  background-color: #f8f8f8;
+  background-color: var(--background-card);
```

Comment:
Any specific reason for going with `background-card` here and not `background-color` ?